specification/ai/Azure.AI.Projects/agents/vector_stores/common/main.tsp

```diff
@@ -87,3 +87,46 @@ model VectorStoreStaticChunkingStrategyResponse
   /** The options for the static chunking strategy. */
   static: VectorStoreStaticChunkingStrategyOptions;
 }
+
+/** Type of vector storage configuration. */
+union VectorStorageDataSourceAssetType {
+  uri_asset: "uri_asset",
+  id_asset: "id_asset"
+}
+
+/**
+ * The structure, containing Azure asset ID and the asset type of the file used as a data source
+ * the enterprise file search.
+ */
+model VectorStorageDataSource {
```

Comment:
please rename to VectorStoreDataSource to be consistent with other VectorStore*

Reply:
Fixed.